### DISCUSSION TITLE
X-Plex-Token for external user in Plex

### DISCUSSION THREAD

**Discussion by karols-lab** [arabcoders/watchstate#571] [General] [answered]
Hi, is there any procedure to extract X-Plex-Token for user that has their own account in Plex, and is invited to my server?

I've tried standard procedure, and for such user there is no option to Get XML in Media Info.

I also tried some method I've looked up in other discussion, with Chrome, Inspect and viewing Network Payload, but Watchstate says that such Token is invalid.

Perhaps I am missing something obvious?

**Comment by arabcoders** [comment]
Hi, i am not sure if plex changed anything, please try the methods described in their official page

https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/

**Reply by karols-lab** [reply]
Thanks, that's what I wrote about - there is no "View XML" on any non-admin account, especially on those external accounts (that have access to Plex server as a friend). Please see screenshot below:

<img width="741" alt="Screenshot 2024-12-12 at 21 45 13" src="https://github.com/user-attachments/assets/76634147-e99b-4c19-8763-1d04114cfe5d" />

**Reply by arabcoders** [reply, marked as answer]
> Thanks, that's what I wrote about - there is no "View XML" on any non-admin account, especially on those external accounts (that have access to Plex server as a friend). Please see screenshot below:
> 
> 
> 
> <img width="741" alt="Screenshot 2024-12-12 at 21 45 13" src="https://github.com/user-attachments/assets/76634147-e99b-4c19-8763-1d04114cfe5d" />
> 
> 

hmmm, it will show up in the network requests. try extracting it and use this script to test it

https://github.com/arabcoders/watchstate/blob/master/FAQ.md#for-plex

if it works we may be able to do something otherwise, we have to somehow support plex authentication via username password like jellyfin/emby
